pkg/storkctl/clusterpair.go

```diff
@@ -887,3 +916,24 @@ func getHostPortFromEndPoint(ep string, ioStreams genericclioptions.IOStreams) (
 func getMissingParameterError(param string, desc string) error {
 	return fmt.Errorf("missing parameter %q - %s", param, desc)
 }
+
+func createNamespace(config string, namespace string) error {
+	conf, err := getConfig(config).ClientConfig()
+	if err != nil {
+		util.CheckErr(err)
```

Comment:
The caller is going to print this error, so we can skip it here?

Reply:
Correct, I removed from this function.